proto/cmp/services/accommodation/v1/info.proto

```diff
@@ -13,6 +13,7 @@ message AccommodationProductInfoRequest {
   cmp.types.v1.RequestHeader header = 1;
 
   // Only respond with the products that are modified after this timestamp
+  // For any on-chain transactions, if applicable, nanoseconds will be ignored
   google.protobuf.Timestamp modified_after = 2;
```

Comment:
This would render in the same line on the buf.build and it would be weird and confusing both because it will look like a single sentence and also the `modified_after` field is not an on-chain operation.

I suggest to update it as below with one extra line between the description of the field and the explanation of the timestamps:

```suggestion
  // Only respond with the products that are modified after this timestamp
  //
  // Timestamps may be used for both off-chain and on-chain operations.
  // For on-chain operations, only seconds are supported, and nanoseconds 
  // will be ignored.
  google.protobuf.Timestamp modified_after = 2;
```

If that's good, it should be applied all of the other changes.